package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "require-podcast",
+  "name": "last-train-podcast",
   "private": true,
-  "description": "Przemyślenia dwójki programistów - Adama i Artura - na temat nowości i wydarzeń w świecie frontendu, backendu i nie tylko.",
+  "description": "Sincere. Honest. Candid.",
   "version": "1.0.0",
```

Comment:
Let's change the version to 0.0.1? and follow smvar versioning rules. 
what do you think?